release/artifacts.yaml

```diff
@@ -159,7 +159,9 @@
   type: jar-scala
 - target: //ledger/sandbox-common:sandbox-common-ce
   type: jar-scala
-- target: //ledger/sandbox-classic:sandbox-classic-ce
+- target: //ledger/sandbox-classic:sandbox-classic
```

Comment:
I'd love to see this one go completely from the published artifacts.
It is probably better to do it in a separate PR though.